README.md

```diff
@@ -370,13 +405,41 @@ Searches logs for a specific service within a time range.
     - `limit` (optional) - Maximum number of logs to return (default: 100)
 
 #### `signoz_get_trace_field_values`
```

Comment:
ah sorry forgot to push my commit that expands on this

Reply:
https://github.com/SigNoz/signoz-mcp-server/pull/49/commits/294b42aedfdcc732271437ee34f42da6948f4928